src/builtins/math.rs

```diff
@@ -62,6 +76,23 @@ fn parse_cmd_opts(
                     opts.scale = scale as usize;
                 }
             }
+            'm' => {
+                let optarg = w.woptarg.unwrap();
+                if optarg.eq(utf32str!("trunc")) {
```

Comment:
No need to make this short, the options should be "truncate" and "ceiling" ("floor" and "round" are fine, of course).